src/components/CreateWallet.jsx

```diff
@@ -60,8 +61,8 @@ const WalletCreationForm = ({ createWallet, isCreating }) => {
 const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
   const [userConfirmed, setUserConfirmed] = useState(false)
 
-  const onSwitch = (e) => {
-    setUserConfirmed(e.target.checked)
+  const onToggle = (toggled) => {
```

Comment:
NIT: `toggle` -> `isToggled` - see [JavaScript Naming Conventions](https://www.robinwieruch.de/javascript-naming-conventions/)

```suggestion
  const onToggle = (isToggled) => {
```

Reply:
💯